dwave/cloud/client/base.py

```diff
@@ -749,7 +749,7 @@ def _fetch_solvers(self,
                          filter=dynamic_fields, no_cache_=True)}
 
             for solver in solvers:
-                if avg_load := loads.get(solver.id) is not None:
+                if (avg_load := loads.get(solver.id)) is not None:
```

Comment:
I am guessing that this means that we're falling back to using the id as the sort key. But setting `id` to `avg_load` might be worth a comment

Reply:
No, we're not -- loads is a mapping `id -> avg_load`. Unless I misunderstood you above.

Reply:
Right, but why `id` -> `avg_load`? It doesn't especially matter I just find that mapping a bit confusing out-of-context

Reply:
Because we're now fetching solvers metadata in two requests -- one for static data (cached), see `solvers` variable above; and one for dynamic data (currently not cached at all), see `loads` variable above.

Dynamic data currently is *only* `avg_load`. So we have this mapping, one `avg_load` for each solver `id`.

This loop here is going through all solvers data (static at this point), and tries to attach dynamic (`avg_load`) to each solver.

Reply:
Makes sense, thanks!